AxMiscFunctions/data/tables/misc-sc-functions-sct.tbm

```diff
@@ -356,11 +356,9 @@ function GetAllPositions()
 
 	local t = {}
 
-	for i=1, #mn.Ships do
+	for thisShip in mn.getShipList() do
 	
 		t[i] = {}
-	
-		local thisShip = mn.Ships[i]
 		t[i].Ship = thisShip.Name
 		t[i].Position = "X:" .. thisShip.Position.x .. ", Y:" .. thisShip.Position.y .. ", Z:" .. thisShip.Position.z
 		t[i].Orientation = "P:" .. thisShip.Orientation.p .. ", B:" .. thisShip.Orientation.b .. ", H:" .. thisShip.Orientation.h
```

Comment:
Since you changed the for loop, is this t[i] still valid here?

Reply:
Whoops, good catch.  Pushed a rewrite.